packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/CustomLineHeightSpan.java

```diff
@@ -54,10 +54,13 @@ public void chooseHeight(
 
       // Round up for the negative values and down for the positive values  (arbitrary choice)
       // So that bottom - top equals additional even if it's an odd number.
-      fm.top -= Math.ceil(additional / 2.0f);
-      fm.bottom += Math.floor(additional / 2.0f);
-      fm.ascent = fm.top;
-      fm.descent = fm.bottom;
+      final int top = (int) (fm.top - Math.ceil(additional / 2.0f));
```

Comment:
Well, as I look at it now, the `(int)` conversion could be put inside the parens for literal 1:1 equivalence